scripts/histmakers/mw_with_mu_eta_pt.py

```diff
@@ -451,6 +497,22 @@ def build_graph(df, dataset):
         results.append(nominal)
         results.append(df.HistoBoost("nominal_weight", [hist.axis.Regular(200, -4, 4)], ["nominal_weight"], storage=hist.storage.Double()))
 
+        if isZveto and auxiliary_histograms:
```

Comment:
Perhaps these utility histograms for the mud veto checks should be removed from this PR. I am not even sure we want to keep these histograms in general, even though they can be disabled with the option for auxiliary histograms.

Reply:
Yes I agree, better to remove